libs/trades/src/lib/trades-table.spec.tsx

```diff
@@ -4,6 +4,18 @@ import { SELL_CLASS, TradesTable, BUY_CLASS } from './trades-table';
 import type { Trade } from './trades-data-provider';
 import { Side } from '@vegaprotocol/types';
 
+const timezoneMock = function (zone: string) {
```

Comment:
This should probably use a beforeAll and you also should add an afterAll to clear the mock